services/static.js

```diff
@@ -269,7 +316,9 @@ Static.getStaticImage = function(config) {
 
 function encodePosition(position) {
   if (position === 'auto') return 'auto';
-
+  if (position.bbox instanceof Array) {
+    return JSON.stringify(position.bbox);
+  }
```

Comment:
[nit] this is a great, but technically redundant given the above assertion you have on L211 that bbox must always be an array of numbers. I think we can simplify this to 

```js
if (position.bbox) {
  return JSON.stringify(position.bbox)
}
```

And then add some error handling above to catch possible problems with the bbox (e.g. too many coordinates).

Reply:
Sounds good.